_i18n/ar/resources/user-guides/make-payment.md

```diff
@@ -0,0 +1,80 @@
+
+9) Enter your wallet password and click on `Send transaction` button.
+
+<img src="/img/resources/user-guides/en/make-payment/password.png" style="width: 427px;"/>
+
+10) Next, you will see the `Transaction successfully sent!` dialog.
+
+<img src="/img/resources/user-guides/en/make-payment/tx_successful.png" style="width: 600px;"/>
+
+This means your transaction was sent to the Monero network and is now waiting to be confirmed (included in a @block) by a miner. Now, you just have to wait for your transaction to be confirmed, which takes on average 2 minutes. Once you've sent your transaction to the network, there is nothing you can do to make the confirmation process faster.
+
+This dialog will also display the `Transaction ID` (txid), which is a unique identifier for your transaction. Optionally, click on `Copy` button to copy it to the clipboard.
+
+Click on `View progress` button to open `Transactions` page and see the transaction details. On this page you can monitor the @blockchain and check if your transaction was confirmed.
+
+### How to check if your payment transaction was confirmed
+After clicking on `View progress` button, `Transactions` page will open with your latest transaction selected.
+
+<img src="/img/resources/user-guides/en/make-payment/transactions_page.png" style="width: 600px;"/>
+
+There are three possible states for your sent transaction:
+- **Sent (Pending)**: your transaction was sent to the Monero network but it is still waiting to be confirmed. Please wait for a while until a miner includes your transaction in a block, confirming it. This process takes on average 2 minutes, but sometimes can take longer.
+- **Sent**: your transaction was included in a block by a miner and is now considered confirmed. On `Blockheight` you can see the block number in which your transaction was included. On `Confirmations` you can see how many blocks were mined since your transaction was included in a block.
+- **Sent (Failed)**: something went wrong and your transaction was not sent to the Monero network. Read the following section to see how you can solve this problem.
+
+### What should I do if my transaction is not confirmed or has failed?
+
+First of all, don't panic. In Monero your money can't get lost "in limbo". If you sent a transaction and it is not getting confirmed or has failed, your money is not at risk of being lost. The money is not removed from your wallet and sent somewhere before arriving at the recipient wallet. There are only two possibilities: either the transaction hasn't been confirmed and the money is still yours, or it was confirmed and the recipient now has the money.
+
+First, try following [this guide](https://monero.stackexchange.com/questions/6649/transaction-stuck-as-pending-in-the-gui). If it doesn't solve your problem, ask for support on [reddit (r/monerosupport)](https://www.reddit.com/r/monerosupport/) or [Monero IRC channels]({{site.baseurl}}/community/hangouts/).
```

Comment:
Better use something else instead of `Monero IRC channels`, since matrix channels are listed as well on that page.

Reply:
"Monero public chat rooms"?